.github/pull_request_template.md

```diff
@@ -21,6 +21,12 @@ Any of the following keywords can be used: close, closes, closed, fix, fixes, fi
 
 (If CI test fails due to known issue, please specify the issue and test PR locally. Then copy & paste the result of "mvn test" to here.)
 
+### Changes that Break Backward Compatibility (Optional)
+
+- My PR contains changes that break backward compatibility or previous assumptions for certain methods or API. They include:
```

Comment:
There should be a checkbox `[ ]` after `-`. Otherwise, it's gonna be be default "breaking backward compatibility"...

Reply:
Sorry for the late response @pkuwm . If we take a look at the Documentation section, it's also optional, and not filling it in means "no documentation"; similarly, not filling this section means "no backward compatibility breaking changes".

Reply:
It seems the dot `-` is by default checked. See the below style format sections. But now worry. Minor. I'll get them fixed together.